tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -38,6 +36,22 @@ def task(event, context):    #pylint: disable-msg=unused-argument
         Raises:
             ExtractFilePathsError: An error occurred parsing the input.
     """
+    LOGGER.debug(event)
+    try:
+        config = event['config']
+        collection = config.get(CONFIG_COLLECTION_KEY, {})
+        exclude_file_types = collection.get(COLLECTION_META_KEY, {}).get(EXCLUDE_FILE_TYPES_KEY, [])
+        if len(exclude_file_types) == 0:
+            LOGGER.info(f"exclude_files_types list is empty.")
+        else:
+            LOGGER.info(f"collected excluded file types {exclude_file_types} from the collection in event.")
+    except KeyError:
+        message = f"arguments are missing from {config}"
```

Comment:
I don't think this will work with Cumulus Logger. You may need to use the older style since `config` is a dictionary. You may also want to get the key it was trying.
```
except KeyError as ke:
        message = "Key {key} is missing from the event configuration: {config}"
        LOGGER.error(message, key=ke, config=config)
        raise KeyError(message.format(key=ke, config=config))
```